pandasai/__init__.py

```diff
@@ -323,9 +323,14 @@ def _is_unsafe_import(self, node: ast.stmt) -> bool:
                 if alias.name in WHITELISTED_OPTIONAL_LIBRARIES:
                     import_optional_dependency(alias.name)
                     continue
-                if not any(
-                    alias.name.startswith(whitelisted_lib)
-                    for whitelisted_lib in WHITELISTED_LIBRARIES
+                if (
+                    alias.name not in WHITELISTED_LIBRARIES
+                    and alias.name != "pandas"
+                    or any(
+                        "." in alias.name
+                        and alias.name.rsplit(".", 1)[0] == whitelisted_lib
+                        for whitelisted_lib in WHITELISTED_LIBRARIES
+                    )
```

Comment:
I believe we can one-line this condition using: `if alias.name.split(".")[0] not in WHITELISTED_LIBRARIES:`

Reply:
yes, no need for additional check for '.' since we only care about the first element